Dockerfile

```diff
@@ -1,6 +1,9 @@
+FROM google/dart:2.5 as dart2
+
 FROM google/dart:1.24.3
 WORKDIR /build/
 ADD pubspec.yaml /build/
-RUN pub get
+COPY --from=dart2 /usr/lib/dart /usr/lib/dart2
+RUN _PUB_TEST_SDK_VERSION=1.24.3 /usr/lib/dart2/bin/pub get --no-precompile
```

Comment:
Usually we do a second `pub get` (using the Dart 1 pub solver) immediately after this. Should we do that here?

Reply:
I thought we need to do second `pub get` only if we run into trouble with `pub2get` when it's unable to resolve `async` dependency. In this repo it resolves correctly, so I would say it's necessary here 🤷‍♀